app/src/main/java/org/jellyfin/androidtv/util/apiclient/BaseItemUtils.kt

```diff
@@ -0,0 +1,231 @@
+package org.jellyfin.androidtv.util.apiclient
+
+import android.content.Context
+import android.text.format.DateFormat
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.TvApp
+import org.jellyfin.androidtv.data.model.ChapterItemInfo
+import org.jellyfin.androidtv.ui.livetv.TvManager
+import org.jellyfin.androidtv.util.TimeUtils
+import org.jellyfin.androidtv.util.Utils
+import org.jellyfin.apiclient.interaction.ApiClient
+import org.jellyfin.apiclient.model.dto.BaseItemDto
+import org.jellyfin.apiclient.model.dto.BaseItemPerson
+import org.jellyfin.apiclient.model.dto.BaseItemType
+import org.jellyfin.apiclient.model.dto.ImageOptions
+import org.jellyfin.apiclient.model.entities.ImageType
+import org.jellyfin.apiclient.model.entities.LocationType
+import org.jellyfin.apiclient.model.entities.PersonType
+import org.jellyfin.apiclient.model.library.PlayAccess
+import org.jellyfin.apiclient.model.livetv.SeriesTimerInfoDto
+import org.koin.java.KoinJavaComponent.get
+import java.util.*
+
+object BaseItemUtils {
```

Comment:
Probably not needed. Adding a `@JvmName` to the top of the file accomplishes the same while creating top-level 
 functions for Kotlin code.